src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/responses/GameResponse.kt

```diff
@@ -1,5 +1,8 @@
 package com.gamingbacklog.api.gamingbacklogapi.models.responses
 
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
added this so the response body doesn't show null (for cleanliness, no functional difference